R/utilities-calculate-pk-parameters.R

```diff
@@ -55,3 +55,354 @@ plotMeanPKParameters <- function(structureSet,
     tables = list(pkAnalysis = pkParametersTable)
   ))
 }
+
+
+#' @title plotPopulationPKParameters
+#' @description Plot PK parameters box plots and tables
+#' @param structureSets `SimulationStructure` R6 class object
+#' @param logFolder folder where the logs are saved
+#' @param settings list of settings for the output table/plot
+#' @param workflowType workflowType Type of population workflow.
+#' Use enum `PopulationWorkflowTypes` to get list of workflow types.
+#' @param xParameters list of parameters to be plotted along x axis
+#' @param yParameters list of parameters to be plotted along y axis
+#' @return list of plots and tables with summary of PK parameters
+#' @export
+#' @import ospsuite
+#' @import tlf
+#' @import ggplot2
+plotPopulationPKParameters <- function(structureSets,

```

Comment:
This is a very, very complicated function!
* 250 lines
*  lots of loops: 3 or 4 imbricated for (hard to tell because I have to scroll so much)

Let split it in meaningful chunks